s3-import-commons/src/test/java/no/unit/nva/publication/s3imports/FileEntriesEventEmitterTest.java

```diff
@@ -51,9 +54,17 @@
 
     public static final String UNEXPECTED_DETAIL_TYPE = "unexpected detail type";
 
+    public static final String INPUT_S3_HOST = "s3:/";
+    public static final String INPUT_BUCKET = "/bucket";
+    public static final String INPUT_PATH = "/parent/folder";
+    public static final String INPUT_FILENAME = "/location.file";
+    public static final String ERROR_FILE = "/location.error";
+    public static final String PARTIAL_FAILURE = "/PartialFailure";
+    public static final String NONE_EXISTING_ERROR_FILE = "/nonexisting.error";
+
```

Comment:
This is in general a bit problematic because we include the path separator to the filenames